.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -76,6 +76,21 @@ jobs:
         with:
            submodules: true
 
+      - name: Output image tag
+        id: image_tag
+        run: |
+          echo ::set-output name=image_tag::$(grep stackhpc_${{ inputs.os_distribution }}_${{ inputs.os_release }}_overcloud_host_image_version etc/kayobe/environments/ci-aio/stackhpc-ci.yml | awk '{print $2}')
```

Comment:
set-output is deprecated, need to switch to $GITHUB_OUTPUT, e.g. 622aee90c79beacd12a03fcf1475d035c7d96174

Reply:
Done